docs/overview.md

```diff
@@ -0,0 +1,34 @@
+# ScalarDB Overview
+
+This page describes what ScalarDB is and its primary use cases.
+
+## What is ScalarDB?
+
+ScalarDB is a cross-database HTAP engine. It achieves ACID transactions and real-time analytics across diverse databases to simplify the complexity of managing multiple databases. Existing solutions are designed to run only transactions or analytical queries across heterogeneous databases, or they run transactions and analytical queries only on homogeneous databases.
```

Comment:
Good point.
Let me think about how I can improve the sentences.
But, let's first publish it and crystalize it later since something is better than nothing.